py3status/py3.py

```diff
@@ -679,3 +702,62 @@ def threshold_get_color(self, value, name=None):
         setattr(self._py3status_module, color_name, color)
 
         return color
+
+    def get_url(self, url):
+        """
+        Get the HTTP Response for the HTTP GET request to `url`.
+
+        Raises `Py3.URLError` on Connection Errors.
+        Raises `Py3.HttpClientError` on HTTP 4xx Errors.
+        Raises `Py3.HttpServerError` on HTTP 5xx Errors.
+        """
+        try:
+            response = urlopen(url)
+        except URLError as e:
```

Comment:
I think we need to worry about more Exceptions being thrown here.

But maybe we will improve this as we go.  I think some tests might be nice at some point.